content/docs/tutorials/get-started/data-pipelines.md

```diff
@@ -137,59 +137,167 @@ stage as <abbr>dependencies</abbr> of another one, we can describe a sequence of
 commands that gets to a desired result. This is what we call a _data pipeline_
 or [_dependency graph_](https://en.wikipedia.org/wiki/Directed_acyclic_graph).
 
-Let's create a second stage chained to the outputs of `prepare.dvc`, to perform
-feature extraction. And a third one for training a machine learning model, based
-on the features:
+Let's create a second stage chained to the outputs of `prepare`, to perform
+feature extraction:
 
 ```dvc
-$ dvc run -f featurize.dvc \
+$ dvc run -n featurize \
+          -p featurize.max_features,featurize.ngrams \
           -d src/featurization.py -d data/prepared \
           -o data/features \
-          python src/featurization.py data/prepared data/features
+          python src/featurization.py \
+                 data/prepared data/features
+```
+
+The `dvc.yaml` is updated automatically and should include two stages now.
+
+And should look like this:
+
+<details>
+
+### 💡Expand to see what happens under the hood
+
+The changes to the `dvc.yaml` should look like this:
+
+```diff
+stages:
+  prepare:
+    cmd: python src/prepare.py data/data.xml
+    deps:
+    - data/data.xml
+    - src/prepare.py
+    params:
+    - prepare.seed
+    - prepare.split
+    outs:
+    - data/prepared
++  featurize:
++    cmd: python src/featurization.py data/prepared data/features
++    deps:
++    - data/prepared
++    - src/featurization.py
++    params:
++    - featurize.max_features
++    - featurize.ngrams
++    outs:
++    - data/features
+```
+
+</details>
+
+<details>
+
+### ⚙️ Expand to add more stages
+
+Let's add the training itself. Nothing new this time, the same `dvc run` command
+with the same set of options:
 
-$ dvc run -f train.dvc \
+```dvc
+$ dvc run -n train \
+          -p train.seed,train.n_estimators \
           -d src/train.py -d data/features \
           -o model.pkl \
           python src/train.py data/features model.pkl
 ```
 
-This would be a good point to commit the changes with Git. This includes any
-`.gitignore` files, and `dvc.yaml` — which describes our pipeline.
+Please, check the `dvc.yaml` again, it should have plus one stage now.
+
+</details>
 
-> 📖 See also the `dvc pipeline` command.
+This should be a good point to commit the changes with Git. This include
+`.gitignore`, `dvc.lock`, and `dvc.yaml` — which describes our pipeline.
 
 ## Reproduce
 
-Imagine you're just cloning the <abbr>repository</abbr> created so far, in
-another computer. It's extremely easy for anyone to reproduce the result
-end-to-end, by using `dvc repro`.
+The whole point of creating this `dvc.yaml` pipeline file is an ability to
+reproduce it:
+
+```dvc
+$ dvc repro
+```
 
 <details>
 
-### 👉 Expand to simulate a fresh clone of this repo
+### ⚙️ Expand to make some fun with it
```

Comment:
```suggestion
### ⚙️ Expand to have some fun with it
```
🙂

Reply:
I like this title but so is this whole expandable section optional? Seems to me it has lots of key info.

Reply:
for get started it's not the point to be very verbose I think, but we can iterate on this later